### PR TITLE
ci: post initium service URL to PR

### PR DESCRIPTION
**Pull request by Pehesi97**

### CODE REVIEW

File: assets/github/onbranch.tmpl [hunk 1]
```diff
@@ -30,6 +32,18 @@ jobs:
...
           INITIUM_CLUSTER_CA_CERT: {{ `${{ secrets.CLUSTER_CA_CERT }}` }}
 
+      - name: comment URL on the PR
+        uses: thollander/actions-comment-pull-request@v2
+        with:
+          comment_tag: initium_deploy_url
+          message: |
+            This Pull Request was deployed to your cluster! 🎉
+
+            | Commit SHA: | {{ `${{ github.event.pull_request.head.sha }}` }} |
```
Comment (francardoso93) [inline assets/github/onbranch.tmpl:42]: `{{` characters are printed on the comment

File: assets/github/onbranch.tmpl [hunk 2]
```diff
@@ -30,6 +32,18 @@ jobs:
...
+            |-------------|----------|
+            | Commit time:  | {{ `${GITHUB_SHA::8}` }} |
```
Comment (francardoso93) [inline assets/github/onbranch.tmpl:44]: `${GITHUB_SHA::8}` isn't replaced by the proper value

File: assets/github/onbranch.tmpl [hunk 3]
```diff
@@ -30,6 +32,18 @@ jobs:
...
+            | URL:        | {{ `${{ steps.deploy.outputs.INITIUM_OUTPUT_URL }}` }} |
```
Comment (francardoso93) [inline assets/github/onbranch.tmpl:45]: This output is empty

**Comment by francardoso93** [comment]
Test result at https://github.com/francardoso93/initium-nodejs-demo-app/pull/4

**Comment by mablanco** [comment]
Superseded by https://github.com/nearform/initium-cli/pull/148

**Pull request closed without merging**